grammar.js

```diff
@@ -336,7 +336,7 @@ module.exports = grammar({
       prec.left(
         seq(
           alias($._dot_without_leading_whitespace, $.dot),
-          $.lower_case_identifier
+          $._lower_case_identifier_without_leading_whitespace
```

Comment:
Matches starting with `_` are anonymous (not named) so they will not be in the normal syntax tree. We probably want something like 
```suggestion
          alias(
            $._lower_case_identifier_without_leading_whitespace,
            $.lower_case_identifier
          )
```

Reply:
Ah, okay. Thanks!